metabrainz/static/css/main.less

```diff
@@ -211,6 +211,26 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong { color: @green; }
+  .people { border-bottom: 2px solid @green; }
```

Comment:
Singular version of a name is probably more appropriate in this case. And I would inherit styling of `hr` element there.